lib/bakeAmbientOcclusion.js

```diff
@@ -902,6 +877,25 @@ function processPrimitive(primitive, meshPrimitiveID, parameters, node) {
 
 ////////// Rendering //////////
 
+function TrianglePixelMarchOptions() {
+    this.uv0 = {};
```

Comment:
Why set these to an empty object?  Why not set them to their actual type, e.g., `new Cartesian3()`?